.github/workflows/annotation_workflow.yml

```
@@ -0,0 +1,23 @@
+name: Specifications Annotation Workflow
```

```suggestion
name: Show Specification Annotations
```

And maybe rename the file to `show_specification_annotations.yml`.